ast/parser_new.go

```diff
@@ -1385,6 +1385,7 @@ func (p *Parser) doScan(skipws bool) {
 
 		p.s.loc.Row = p.s.pos.Row
 		p.s.loc.Col = p.s.pos.Col
+		p.s.loc.Offset = p.s.pos.Offset
```

Comment:
We still use `p.s.pos.End` in a few spots, but I can change to only keep around the end + location rather than the full position (save us a few int's per save 😄)

Reply:
Let's not worry about that for now until we do some more performance analysis.

Reply:
Was a quick `sed` sorta change, and gave ~5% improvement on memory usage so I figured why not